### ISSUE TITLE
Make a KNX read request 

### ISSUE THREAD

**Issue by staehler** [openhab/openhab#1456]
As KNX status messages are not reliable (esp. with KNX powernet), there is a need to force in some cases to read the KNX state of an item.
This feature is implemented when openhab starts up, but it is not possible to read again when openhab is already running.
Please add this feature!

Thanks in advance,
Michael

**Comment by maxpg** [comment]
Is there a possibility to force a status update from the rule. I would like to force an update from the rules, depending on the status of an item. So if it is uninitialized I would like to force an update. But I don't want to have updates for all of my contacts regularly.

**Comment by Snickermicker** [comment]
Wouldn't "<" do the trick for you? That was meant to read once from the bus to initialize the DP on startup.
If you supply a time like 3600 then an initial read is performed and after 3600 the timer reads again.

**Comment by maxpg** [comment]
In general yes. But if there is a timeout on the initial read for example, then the item is never initialized...

**Comment by Snickermicker** [comment]
Sorry for beeing a pest, but if the initial read fails, then reading is  retried 2 more times.
If the device is not readable during his init phase, then the scheduler would trigger again when the timer expires (if configured).

**Comment by maxpg** [comment]
Ok, to be honest, I am not sure how I got to the situation. I have done quite some rule/item changes during the last 2 weeks. It happened twice to me that all of my window contacts were in state "undefined" (so my automatic rollershutter program did not work as expected). I have configured the items with the "<" and it generally works. So after loading the items again (added a blank to the items file and saved it again) all states were initialized fine.

I cannot reproduce how this happened. I just thought if I could create a rule/config that runs from time to time and reads again all items that have the state "uninitialized" it would be more robust. But yes, this would be some kind of a workaround for the problem that I don't have really identified until now.
